library/src/blas3/trtri_device.h

```diff
@@ -106,7 +106,7 @@ trtri_device(rocblas_fill uplo,
         //use the diagonal one to update current column
         if(tx > col) reg += sA[tx + col * n] * sA[col + col * n];
 
-        __syncthreads(); // since NB < 64, this synch can be avoided
+        __syncthreads(); // if NB < 64, this synch can be avoided
```

Comment:
should line 120 and line 126 also say "if NB < 64" in place of "since NB < 64" ?